src/AutoUpdate/Hooks.php

```diff
@@ -21,17 +21,30 @@ class Hooks implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC
 	/** @var \WPML_Translation_Element_Factory $elementFactory */
 	private $elementFactory;
 
+	private $postsToRefreshNeedsUpdate;
+
 	public function __construct(
 		\WPML_PB_Integration $pbIntegration,
 		\WPML_Translation_Element_Factory $elementFactory
 	) {
-		$this->pbIntegration     = $pbIntegration;
-		$this->elementFactory    = $elementFactory;
+		$this->pbIntegration             = $pbIntegration;
+		$this->elementFactory            = $elementFactory;
+		$this->postsToRefreshNeedsUpdate = wpml_collect();
```

Comment:
@strategio where this is used?

Reply:
Good catch @andreasciamanna! This is a leftover, I removed it :)